x-pack/plugins/apm/common/rules/apm_rule_types.ts

```diff
@@ -40,6 +40,11 @@ export enum AggregationType {
   P99 = '99th',
 }
 
+export interface PreviewChartResponseItem {
+  name: string;
+  data: Array<{ x: number; y: number | null }>;
+}
+
```

Comment:
Can you move this to `x-pack/plugins/apm/server/routes/alerts/route.ts`. If you want to import it from client or else where you can still do that with something like `import type { PreviewChartResponseItem } from '../../../server/routes/alerts/route' `

Reply:
updated